### PR TITLE
fix bug 1173122 - fix footer license link

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -155,7 +155,7 @@
       <div class="column-main">
         {% block footer_copyright_redesign %}
         <p>
-          {% trans copyright_url=wiki_url('Project:MDN/About#Copyrights_and_licenses'), about_url=wiki_url('Project:MDN/About'), thisyear=thisyear() %}
+          {% trans copyright_url='/docs/MDN/About#Copyrights_and_licenses', about_url='/docs/MDN/About', thisyear=thisyear() %}
```
Comment (jezdez) [inline templates/base.html]: When opening the URL `/docs/MDN/About#Copyrights_and_licenses` I get redirected to the `en-US` version but the hash is lost in the redirect :(

**Comment by jezdez** [comment]
rwc+

**Comment by groovecoder** [comment]
Updated to use `wiki_url` which uses the locale prefixer to avoid the redirect that loses the url hash.

**Comment by jezdez** [comment]
r+

**Pull request merged**